backend/server/tests/test_validation.py

```diff
@@ -0,0 +1,41 @@
+# we want to assure that courses that may be accessed from a container are always accurately computed.
+from contextlib import suppress
+import requests
+
+from server.tests.test_getAllUnlocked import USERS
+
+# TODO: Do we care if these courses are broken before deployment? they are the honours courses + some wierd courses
```

Comment:
this is an important note

Reply:
If you're going to include them in production, would it be possible to somehow alert users that the courses they have selected are problematic?

Reply:
yup @lhjt we have an indication in the frontend that we cant parse them